index.js

```diff
@@ -62,22 +76,27 @@ conventionalRecommendedBump({
     console.error(chalk.red(err.message))
     return
   }
-
   var newVersion = pkg.version
   if (!argv.firstRelease) {
-    newVersion = semver.inc(pkg.version, release.releaseAs)
+    newVersion = getNextVersion(release)
     checkpoint('bumping version in package.json from %s to %s', [pkg.version, newVersion])
     pkg.version = newVersion
     fs.writeFileSync(pkgPath, JSON.stringify(pkg, null, 2) + '\n', 'utf-8')
   } else {
     checkpoint('skip version bump on first release', [], chalk.red(figures.cross))
   }
-
-  outputChangelog(argv, function () {
+  // only commit the package.json on prerelease
```

Comment:
@bcoe I thought about combining the pre-release logs to a single one, but what about pre-releases containing fixes to other pre-releases. e.g. You do a pre-release of a new feature as `v2.1.0-rc.0` with `feat(god): can now obtain god powers`, but then fix an edge case of this new feature in the next prerelease `v2.1.0-rc.1`: `fix(god): fix edge case to give full god instead of demigod powers`. Both of them end up in the combined `v2.1.0` changelog and it might end up with quite a lot of 'fixes inside of this release' that are out of context.